core/src/it/java/org/pentaho/platform/dataaccess/datasource/wizard/service/impl/ConnectionServiceImplIT.java

```diff
@@ -83,8 +82,7 @@ public class ConnectionServiceImplIT {
   private static MockDatasourceMgmtService mgmtService = new MockDatasourceMgmtService();
 
   @BeforeClass
-  public static void setUpClass() throws PlatformInitializationException, DuplicateDatasourceException,
-    DatasourceMgmtServiceException {
+  public static void setUpClass() throws Exception {
```

Comment:
Specific exception declaration be preferred to generic ones. As this is test code, not so relevant, but personally I'd go for always applying the best practices in UTs also.

Reply:
Done.